### DISCUSSION TITLE
Where is the HomeKit QR code for my Amcrest camera(s)?

### DISCUSSION THREAD

**Discussion by Scenario** [koush/scrypted#611] [Q&A] [answered]
In https://github.com/koush/scrypted/wiki/HomeKit-Secure-Video-Setup, Step 6 says:

> In the Camera's Scrypted settings, scan the QR code in the HomeKit app on iOS or Mac.
Do not use the QR code in the the HomeKit Plugin, this plugin is a bridge for bridging non-camera devices like switches, thermostats, etc. Cameras are run in standalone accessory mode by default, and are not bridged.

I can only find the QR code for the HomeKit Plugin. I have my first Amcrest camera configured and I see it's image (static?) but I don't see a different QR code anywhere in the Scrypted UI.

What do you mean, "In the Camera's Scripted settings?" We need an explicit path here.

**Comment by koush** [comment, marked as answer]
<img width="836" alt="image" src="https://user-images.githubusercontent.com/73924/230703280-300c31be-c893-4a15-886f-a95530937753.png">
